config/schema/document_types/asylum_support_decision.json

```diff
@@ -140,40 +140,94 @@
     ],
     "tribunal_decision_category": [
       {
-        "label": "Asylum Seeker, Definition of",
-        "value": "asylum-seeker-definition-of"
+        "label": "Section 95 (asylum-seekers)",
+        "value": "section-95-asylum-seekers"
       },
       {
-        "label": "Breach of Conditions by:",
-        "value": "breach-of-conditions-by"
+        "label": "Section 4(2) (failed asylum seekers)",
+        "value": "section-4-2-failed-asylum-seekers"
       },
       {
-        "label": "European Convention of Human Rights",
-        "value": "european-convention-of-human-rights"
+        "label": "Section 4(1) (neither an asylum seeker nor a failed asylum seeker)",
+        "value": "section-4-1-neither-an-asylum-seeker-nor-a-failed-asylum-seeker"
+      }
+    ],
+    "tribunal_decision_sub_category": [
+      {
+        "label": "Section 95 - jusrisdiction",
+        "value": "section-95-jusrisdiction"
+      },
+      {
+        "label": "Section 95 - destitution",
+        "value": "section-95-destitution"
+      },
+      {
+        "label": "Section 95 - children",
+        "value": "section-95-children"
+      },
+      {
+        "label": "Section 95 - dependents",
+        "value": "section-95-dependents"
```

Comment:
But @tommyp thats a good spot. They are wrong in the master sheet hence the generator has pulled them across. Will amend.